guided-match/ddl.sql

```diff
@@ -0,0 +1,204 @@
+  threshold_value                   NUMBER(12,2),
+  min_threshold_date                DATE,
+  max_Threshold_date                DATE,
+  threshhold_date                   DATE
+);
+
+*/
+
+/* This table could exist in some form in both databases */
+CREATE TABLE journeys (
+  journey_id                        INTEGER PRIMARY KEY,
+  journey_name                      VARCHAR(200),
+  published_status                  CHAR(1), -- Y(es) or N(o)
+  parent_journey_id                 INTEGER
+);
+CREATE INDEX JOUR_IDX1 on JOURNEYS(journey_name);
+CREATE INDEX JOUR_IDX2 on JOURNEYS(parent_journey_id);
+
+/* Neo4J
+CREATE TABLE lot_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   lot_number                       VARCHAR(10) NOT NULL        
+);
+*/
+
+/* Similar to the commercial_agreements table this one too also belongs in the agreements service
+CREATE TABLE lots (
+  lot_id                            INTEGER PRIMARY KEY,
+  lot_number                        VARCHAR(20) NOT NULL,
+  lot_name                          VARCHAR(200) NOT NULL,
+  commercial_agreement_id           INTEGER NOT NULL,
+  lot_type                          VARCHAR(25) NOT NULL, -- Valid values Services, Products or Products and Services
+  start_date                        DATE NOT NULL,
+  end_date                          DATE NOT NULL
+);
+CREATE INDEX LOTS_IDX1 on LOTS(lot_number);
+CREATE INDEX LOTS_IDX2 on LOTS(lot_name);
+CREATE INDEX LOTS_IDX3 on LOTS(commercial_agreement_id);
+*/
+
+/* Neo4J
+CREATE TABLE question_definitions (
+  question_id                       INTEGER PRIMARY KEY,
+  question_type                     VARCHAR(20) NOT NULL,
+  question_description              VARCHAR(2000) NOT NULL,
+  question_text                     VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL, -- Y(es) or N(o),
+  validate_via_agreement            CHAR(1) NOT NULL, -- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_groups (
+  question_group_id                 INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(100) NOT NULL,
+  published_status                  CHAR(1) NOT NULL -- Need to check if we don't want to make this boolean
+);
+*/
+/* Neo4J
+CREATE TABLE question_group_members (
+  question_group_member_id          INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL-- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_outcome_evaluators (
+  outcome_evaluator_code            VARCHAR(10) PRIMARY KEY,
+  outcome_evaluator_description     VARCHAR(200) NOT NULL
+);
+*/
+
+CREATE TABLE search_domains (
+  domain_modifier_id               INTEGER PRIMARY KEY,
+  search_id                        INTEGER NOT NULL,

```

Comment:
This is a foreign key from `search_terms` presumably - formalise this using `references`